hypertrace-gradle-docker-plugin/src/main/java/org/hypertrace/gradle/docker/DockerPlugin.java

```diff
@@ -109,17 +112,17 @@ private boolean isReleaseVersion(Project project) {
   }
 
   private String getBranchTag() {
-    // Use the value of CIRCLE_BRANCH environment variable if defined (that is, the branch name used
-    // to build in CI), otherwise for local builds use 'test'
-    return getEnvironmentVariable("CIRCLE_BRANCH")
+      // Use the value of CIRCLE_BRANCH/ GHA_BRANCH environment variable if defined (that is, the branch name used
+      // to build in CI), otherwise for local builds use 'test'
+    return getFirstAvailableEnvironmentVariable(Arrays.asList("CIRCLE_BRANCH", "GHA_BRANCH"))
         .map(String::trim)
         .map(branch -> branch.replaceAll("[^A-Za-z0-9\\.\\_\\-]", ""))
         .filter(branch -> !branch.isEmpty())
         .orElse("test");
   }
 
   private Optional<String> tryGetBuildSha() {
-    return getEnvironmentVariable("CIRCLE_SHA1")
+    return getFirstAvailableEnvironmentVariable(Arrays.asList("CIRCLE_BRANCH", "GHA_BRANCH"))
```

Comment:
Why would this be on `BRANCH`? I think we need a similar env for GHA?

Reply:
I think we need `GITHUB_SHA`?

Reply:
`GITHUB_SHA` is already available.

Reply:
reverted to previous function